src/components/Filters/SimpleFilter/SimpleFilter.js

```diff
@@ -0,0 +1,50 @@
+import PropTypes from 'prop-types';
+import { useSelector } from 'react-redux';
+import { selectAggsState } from '../../../reducers/aggs/selectors';
+import { selectQueryState } from '../../../reducers/query/selectors';
+import { coalesce } from '../../../utils';
+import CollapsibleFilter from '../CollapsibleFilter';
+import MoreOrLess from '../MoreOrLess';
+import AggregationItem from '../AggregationItem/AggregationItem';
+
+//TODO: when upgraded to redux toolkit, use createSelector
+//to derive this data directly from aggs and query state
+export const extraData = (fieldName, aggs, query) => {
+  const activeChildren = coalesce(query, fieldName, []);
+  const options = coalesce(aggs, fieldName, []);
+  const hasChildren = activeChildren.length > 0;
+
+  return { options, hasChildren };
```

Comment:
can this go directly into SimpleFilter below?

Reply:
This has been updated and properly verified in unit tests